### DISCUSSION TITLE
Capability Area — Data Architecture - EKG/Maturity

### DISCUSSION THREAD

**Discussion by giscus** [EKGF/ekg-maturity#83] [Page Comments]
# Capability Area — Data Architecture - EKG/Maturity

Maturity Model for the Enterprise Knowledge Graph (EKG)

https://maturity.ekgf.org/pillar/data/data-architecture/

**Comment by jgeluk** [comment]
@rivettp why don't we have "Data Catalogs" here as a topic?
